thirdparty/CMakeLists.txt

```diff
@@ -131,15 +131,16 @@ if(NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_IMPORT_ENABLE)
             set(Protobuf_DEBUG ON)
         endif()
         find_package(Protobuf 3.9.0 REQUIRED)
+        set(Protobuf_LITE_LIBRARIES protobuf::libprotobuf-lite)
         if(NGRAPH_USE_PROTOBUF_LITE)
-            set(Protobuf_LIBRARIES protobuf::libprotobuf-lite)
+            set(Protobuf_LIBRARIES ${Protobuf_LITE_LIBRARIES})
         else()
             set(Protobuf_LIBRARIES protobuf::libprotobuf)
         endif()
         set(SYSTEM_PROTOC protobuf::protoc)
         set(PROTOC_EXECUTABLE ${SYSTEM_PROTOC})
 
-        foreach(target ${SYSTEM_PROTOC} ${Protobuf_LIBRARIES})
+        foreach(target ${SYSTEM_PROTOC} ${Protobuf_LIBRARIES} ${Protobuf_LITE_LIBRARIES})
```

Comment:
Is similar update needed in thirdparty/protobuf/CMakelists.txt?

Reply:
I suppose no, because here we set `IMPORTED_GLOBAL`  which is necessary only for imported targets